package.json

```diff
@@ -21,14 +21,16 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.5.0",
     "body-parser": "^1.16.0",
-    "react": "^15.1.0",
-    "react-dom": "^15.1.0",
-    "webpack": "^1.13.1",
     "express": "^4.14.1",
+    "immutability-helper": "^2.1.1",
```

Comment:
O, this was an oversight from me using a function I needed immutable for, deleting it, and forgetting to delete that srry